documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_6_Usage_Control.md

```diff
@@ -1 +1,263 @@
+
+In order to control the usage of data, one approach can be to intercept the data flow between the services and applications. [Figure 4.1.6.7](https://user-images.githubusercontent.com/69632955/150939429-92add160-5c72-42fa-8b07-6716921caebf.png) shows as example of how developers can do this. Apache Camel offers the possibility to integrate interceptors that it executes every time before and after a processor is working.
+
+As the International Data Spaces provides an Information Model (see [Section 3.1](../../3_Layers_of_the_Reference_Architecture_Model/3_1_Business_Layer/)), additional metadata enhances the data transferred via the route, thereby enabling better Usage Control enforcement. The Connector attaches the metadata to the data package, as explained in [Section 3.4](../../3_Layers_of_the_Reference_Architecture_Model/3_4_Information_Layer/). In addition, a PIP is able to resolve more metadata during the decision-making process if necessary.
+
+![image](https://user-images.githubusercontent.com/69632955/150939429-92add160-5c72-42fa-8b07-6716921caebf.png)
+
+_Figure 4.1.6.7: Intercepting Apache Camel data flows_
+
+This paradigm also works across company borders, as data always flows through the IDS Connector and the Apache Camel interceptor, respectively (as shown in [Figure 4.1.6.8](https://user-images.githubusercontent.com/69632955/150939461-58fc264c-db41-4d40-bad1-e6e73f08d620.png)). When reaching the receiving Connector, the respective policy to protect the data is automatically instantiated.
+
+Depending on the policies available, this way of enforcement is not enough to cover all possible use cases and full Usage Control.
+
+![image](https://user-images.githubusercontent.com/69632955/150939461-58fc264c-db41-4d40-bad1-e6e73f08d620.png)
+
+_Figure 4.1.6.8: Data flow across company borders_
+
+#### **CONTEXT INFORMATION AND OBLIGATION FULLFILLMENT**
+
+Context information is the information such as time, location, states of systems, etc, that is required for the evaluation of a usage policy. In IDS, one can basically distinguish 2 types of information;  the information that is obtained directly from the Connector and the information that is obtained from a global source (see [Figure 4.1.6.9](media/Context-information-and-execution-point-PIP.drawio.png)). The information that can be obtained from the Connector itself, such as a system state, is referred to as local context information. On the other hand, the information that contains, for example, the state of a payment or information from the IDS ParIS is referred to as global context information. The IDS Usage Control language provides ways to address and use these context infomation. For example, They shall be provided by an IDS Policy Information Point (IDS PIP). A Usage Control technology can then use these context information for enforcement by connecting the PIPs.
+
+![image](media/Context-information-and-execution-point-PIP.drawio.png)

```

Comment:
The Policy Engine is part of the IDS Connector core, not a separat thing.

Reply:
not necessarily, it must only be connected.